vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2

```diff
@@ -1,25 +1,16 @@
 [global]
 netbios name = {{ samba_netbios_name }}
 clustering = yes
-passdb backend = tdbsam
 log file = /var/log/samba/log.%m
 max log size = 0
 server string = Samba server version %v
 workgroup = MYGROUP
 security = user
 
-client signing = auto
-
 kernel share modes = No
 map archive = No
-map hidden = no
```

Comment:
don't we want to be explicit about the `map foobar` parameters and `store dos attributes`?

Reply:
For a while we haven't seen any issues with these **map {archive,hidden,system,read only}** parameters. In fact we still haven't explicitly specified `map system = no`(which is the default) in this template. I hope we will be able to notice breakage as soon as some defaults are changed or a regression sneaks in through code.